CONTENTS.md

```diff
@@ -392,7 +397,7 @@ Clones projects and dependencies between OBS instances.
 * Sources: [obs_clone.py](obs_clone.py)
 * Documentation: --
 * Package: openSUSE-release-tools
-* Usage: ???
+* Usage: obsolete
```

Comment:
Maybe mention that was originally intended for testcases, but a different approach for testing was then introduced